rest-api/alembic/versions/ac995f0b54b8_ehr_metrics_use_organizations.py

```diff
@@ -0,0 +1,66 @@
+"""ehr-metrics-use-organizations
+
+Revision ID: ac995f0b54b8
+Revises: 03bdb0a44083
+Create Date: 2019-03-28 14:37:16.010194
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import model.utils
+from sqlalchemy.dialects import mysql
+
+from participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from participant_enums import MetricSetType, MetricsKey
+from model.base import add_table_history_table, drop_table_history_table
+from model.code import CodeType
+from model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = 'ac995f0b54b8'
+down_revision = '03bdb0a44083'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('ehr_receipt', sa.Column('organization_id', sa.Integer(), nullable=False))
+    op.drop_constraint(u'ehr_receipt_ibfk_1', 'ehr_receipt', type_='foreignkey')
```

Comment:
Do you need the columns in a specified order? If yes, we may need to specify the order by using `AFTER`.

Reply:
No the column order does not matter for this table.